apps/omg_status/test/omg_status/metric/datadog_test.exs

```diff
@@ -28,11 +38,10 @@ defmodule OMG.Status.Metric.DatadogTest do
         port = Port.open({:spawn, "cat"}, [:binary])
         true = Process.link(datadog_pid)
         true = Process.exit(port, :portkill)
+
         # we want to exit because the port forcefully closes
         # so this sleep shouldn't happen
         Process.sleep(10_000)
       end)
-
-    assert_receive {:trace, ^datadog_pid, :receive, {:EXIT, port, :portkill}}
```

Comment:
removal of this assertion is a bit suspicious - there's no assertions left here. Also the name of the test indicates we want to test an exit signal.

Reply:
Good catch! This one slipped!